magento/pwa-studio
# [doc] Extension development

## Description

Adds a new doc topic about extension development.

## Related Issue

Closes PWA-797

## Acceptance

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Build the HTML preview server: `yarn develop`
3. Verify site builds and accessible from browser
4. Navigate to `tutorials/extension-development/`
5. Verify topic exists and listed on the left navigation

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
